src/PlaywrightSharp/PlaywrightSharp.csproj

```diff
@@ -1,67 +1,67 @@
-		<DriverFiles Include="runtimes\osx\**">
-			<InProject>false</InProject>
-		</DriverFiles>
-	</ItemGroup>
-	<ItemGroup Condition="$([MSBuild]::IsOSPlatform('Linux'))">
-		<DriverFiles Include="runtimes\unix\**">
-			<InProject>false</InProject>
-		</DriverFiles>
-	</ItemGroup>
-	<ItemGroup>
-		<ContentWithTargetPath Include="@(DriverFiles)" TargetPath="%(RecursiveDir)%(Filename)%(Extension)">
-			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-		</ContentWithTargetPath>
-	</ItemGroup>
-	<Target Name="EnsurePrerequisitsRan" BeforeTargets="InstallBrowsers">
+  <PropertyGroup>
+    <Title>PlaywrightSharp</Title>
+    <PackageId>PlaywrightSharp</PackageId>
+    <Summary>Playwright-Sharp .NET library to automate Chromium, Firefox and WebKit with a single API.</Summary>
+    <Description>PlaywrightSharp is a .NET library to automate Chromium, Firefox and WebKit with a single API. Playwright is built to enable cross-browser web automation that is ever-green, capable, reliable and fast.</Description>
+    <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
+    <DebugType>embedded</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <DocumentationFile>bin\$(Configuration)\netstandard2.0\PlaywrightSharp.xml</DocumentationFile>
+    <RunWithWarnings>true</RunWithWarnings>
+    <StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
+    <CodeAnalysisRuleSet>../PlaywrightSharp.ruleset</CodeAnalysisRuleSet>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <RootNamespace>PlaywrightSharp</RootNamespace>
+    <ReleaseVersion>0.0.0</ReleaseVersion>
+    <BuildFromSource>True</BuildFromSource>
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <NoWarn>1701;1702;CS0067</NoWarn>
+  </PropertyGroup>
+  <Import Project="../Common/Dependencies.props" />
+  <Import Project="../Common/SignAssembly.props" />
+  <Import Project="../Common/PackageInfo.props" />
+  <Import Project="build/netstandard2.0/PlaywrightSharp.targets" />
+  <ItemGroup>
+    <PackageReference Include="Esprima" Version="1.0.1270" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.1" />
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
+    <PackageReference Include="Macross.Json.Extensions" Version="1.5.0" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="playwright\osx\**" Pack="true" PackagePath="playwright\osx\native" />
+    <None Include="playwright\unix\**" Pack="true" PackagePath="playwright\unix\native" />
+    <None Include="playwright\win-x64\**" Pack="true" PackagePath="playwright\win-x64\native" />
+    <None Include="playwright\win-x86\**" Pack="true" PackagePath="playwright\win-x86\native" />
```

Comment:
how about drivers?

Reply:
Sure, but let's do `drivers` for NuGet, and `.playwright` for customer's `bin` folders to avoid potential conflicts.